lib/phoenix_live_view.ex

```diff
@@ -1338,6 +1347,12 @@ defmodule Phoenix.LiveView do
     Phoenix.LiveView.Channel.send_update(pid, module, id, assigns)
```

Comment:
Suggestion: make this line be:

```suggestion
    Phoenix.LiveView.Channel.send_update(pid, {module, id}, assigns)
```

This way you can share more code between both branches and disambiguate only in the internals. :)